README.md

```diff
@@ -250,6 +252,14 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Announcement |
 | --- | --- | --- | --- |
+| [Opyn Gamma Protocol](https://www.opyn.co/) | May 2021 | 6 person-weeks | |
```

Comment:
Drop Protocol from name

Reply:
updated pull request to reflect this change